lib/sentimental.rb

```diff
@@ -1,23 +1,23 @@
+require 'json'
+
 class Sentimental
   attr_accessor :threshold, :word_scores, :neutral_regexps, :ngrams
 
   def initialize(threshold: 0, word_scores: nil, neutral_regexps: [], ngrams: 1)
-    if ngrams >= 1
-      @ngrams = ngrams.to_i
-    else
-      @ngrams = 1
-    end
+    @ngrams = ngrams.to_i if ngrams >= 1
+    @ngrams = 1 unless ngrams >= 1
```

Comment:
I can leave the if else although rubocop complains. What if we put something like:
    
    @ngrams = ngrams.to_i.abs

I believe that not only improves readability but also express the intention of the code (we are expecting integers positive and grater or equal than 1